src/components/Settings/SaveSearchItem.vue

```diff
@@ -46,6 +46,7 @@ export default {
 	name: 'SaveSearchItem',
 	data() {
 		return {
+
 			showAlerts: this.savedSearch?.isAlert,
 			deleteSearch: () => []
```

Comment:
Please remove the extra blank line and this `deleteSearch` key/value pair. It's unused.